src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -378,7 +387,7 @@ private BoundExpression BindNamedAttributeArgument(AttributeArgumentSyntax named
                 var containingAssembly = fieldSymbol.ContainingAssembly as SourceAssemblySymbol;
 
                 // We do not want to generate any unassigned field or unreferenced field diagnostics.
-                containingAssembly?.NoteFieldAccess(fieldSymbol, read: true, write: true);
+                containingAssembly?.NoteFieldAccess(fieldSymbol.OriginalDefinition, read: true, write: true);
```

Comment:
>containingAssembly?.NoteFieldAccess(fieldSymbol.OriginalDefinition, read: true, write: true); [](start = 16, length = 93)

Instead of making this change, we should change implementation of  NoteFieldAccess to dig to the definition. Then all the consumer will benefit from the change.